server/plugin.go

```diff
@@ -15,6 +17,9 @@ const (
 	USER_SURVEY_KEY     = "UserSurvey-"
 
 	SURVEYBOT_DESCRIPTION = "Surveybot collects user feedback to improve Mattermost. [Learn more](https://mattermost.com/pl/default-nps)."
+
+	DEFAULT_UPGRADE_CHECK_MAX_DELAY = 2 * time.Minute
+	DEAFULT_USER_SURVEY_MAX_DELAY   = 1 * time.Second
```

Comment:
This seems unexpectedly short -- presumably we don't need to delivery surveys within 1 second of whatever event triggers this. Why not wait 1 minute and drastically reduce the chances of a collision? (I'm thinking about unexpected delays in the operation to process the user survey.)

Reply:
I could certainly increase it. Since this one is used in response to an HTTP request, I was originally worried about keeping the delay small to avoid holding the connection open, but I moved that into a goroutine because I was still concerned about it even with a max delay of only one second. I'll increase it to a few seconds, but I'd still like this to be relatively responsive, and I think more than that may cause problems. A longer delay will also increase the chance that a user refreshes or does something that'll trigger another check as well.